### PR TITLE
Add option to suggest exact-match GPL identifiers

### PR DESCRIPTION
**Pull request by dabutvin**
Adds an option object to opt out of the GPL upgrades.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -315,24 +316,25 @@ module.exports = function (identifier) {
...
+      'GPL-1.0+', 'GPL-2.0+', 'GPL-3.0+',
+      'LGPL-2.0+', 'LGPL-2.1+', 'LGPL-3.0+',
+      'AGPL-1.0+', 'AGPL-3.0+'
+    ].indexOf(value) !== -1) {
+      return value.replace(/\+$/, '-or-later')
+    } else if (['GPL-3.0', 'LGPL-3.0', 'AGPL-3.0'].indexOf(value) !== -1) {
+      return value + '-or-later'
+    } else {
+      return value
+    }
```
Comment (kemitchell) [inline index.js]: Did the whitespace change in this function? Looks like a one-line insertion.

Comment (dabutvin) [reply index.js]: Yep. I moved this function into the function above so we could reference the options parameter from this one place.

Do you think there is a better way?

**Comment by kemitchell** [comment]
@dabutvin I've pushed a commit to avoid ES6 language features in handling the optional `options` argument, and also avoiding the reindentation of the `upgradeGPLs` helper function. My revised passes all the tests, including your new ones.

I'd be happy to take this as a minor release.

Do you license your work to the public under the terms of Apache 2.0?

**Comment by kemitchell** [comment]
@dabutvin If you're alright licensing Apache 2.0, please push a commit adding yourself to the `contributors` array in `package.json`.

**Comment by kemitchell** [comment]
Published as 3.1.0.

**Comment by kemitchell** [comment]
@dabutvin I've also added you to the jslicense GitHub org. Please open pull request for any more than rote housekeeping changes.

**Pull request merged**